.tours/filepathNavigationSupportTour.tour

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "tourFile": "supportPathInStepFileProperty.tour",
```

Comment:
This property should match the actual file name, so that `StateManager` can handle it properly.
Currently, if I select `Jump to .tour Source` (from Tours tree context menu, it will fail as the file **supportPathInStepFileProperty.tour** does not exist. 
Of course, this could be handled differently by StateManager and will reconsider it the future.

**Note**: This is just a note, as the tour is just for the PR context so there is no need for any change